src/mqueryfront/src/RecentPage.js

```diff
@@ -1,17 +1,182 @@
 import React, { Component } from "react";
 import SearchJobs from "./SearchJobs";
+import ErrorBoundary from "./ErrorBoundary";
+import axios from "axios";
+import { API_URL } from "./config";
 
 class RecentPage extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            jobs: [],
+            filter: null,
+            head: [],
+            activePage: 1,
+            itemsPerPage: 11,
```

Comment:
A bit unintuitive, I'd assume that pages have 10 elements

```suggestion
            itemsPerPage: 10,
```

Reply:
OK.